File: 2020/12/week1/1ilsang/algorithm/leetcode1662.js
/**
 * https://leetcode.com/contest/weekly-contest-216/problems/check-if-two-string-arrays-are-equivalent/
 * 1662. Check If Two String Arrays are Equivalent
 * @param {string[]} word1
 * @param {string[]} word2
 * @return {boolean}
 */
const arrayStringsAreEqual = (word1, word2) => {
  const prev = word1.join("");
  const next = word2.join("");

  return prev === next;
};

File: 2020/12/week1/minhyungPark/algorithm/LeetCode1662.kt
/**
 * Leetcode 1662. Check If Two String Arrays are Equivalent
 * https://leetcode.com/contest/weekly-contest-216/problems/check-if-two-string-arrays-are-equivalent/
 */

class LeetCode1662 {
    fun arrayStringsAreEqual(word1: Array<String>, word2: Array<String>): Boolean {
        return word1.joinToString(separator = "") == word2.joinToString(separator = "")
    }
}